index.html

```diff
@@ -31,9 +31,53 @@ <h1 class="title">Error</h1>
 };
     </script>
 
-    <script src="node_modules/three/build/three.js"></script>
-    <script src="node_modules/three/examples/js/controls/VRControls.js"></script>
-    <script src="node_modules/three/examples/js/effects/VREffect.js"></script>
-    <script src="build/embed.js"></script>
+    <script>
+      (function () {
+        function loadScript(src, options) {
+          var script = document.createElement('script');
+          script.src = src;
+          script.async = false;
+          document.head.appendChild(script);
+
+          if (typeof options !== 'undefined') {
+            script.onload = options.callbackSuccess;
+            script.onerror = options.callbackError;
+          }
+        }
+
+        function loadPageScripts () {
+          var files = [
+            'node_modules/three/build/three.js',
+            'node_modules/three/examples/js/controls/VRControls.js',
+            'node_modules/three/examples/js/effects/VREffect.js',
+            'build/embed.js'
+          ];
+
+          if (!('getVRDisplays' in navigator)) {
+            files.unshift(
+              'node_modules/webvr-polyfill/build/webvr-polyfill.js'
+            );
+          }
+
+          // Load each file. Since async is false, these will be
+          // executed in order.
+          for (var i = 0; i < files.length; i++) {
+            loadScript(files[i]);
+          }
+        }
+
+        // If Promises aren't supported make sure that's loaded before the other
+        // scripts, and defer the other scripts.
+        if (!'Promise' in window) {
+          return loadScript('node_modules/es6-promise/es6-promise.min.js',
+            loadPageScripts, function(err) {
+              console.error(err);
+            });
```

Comment:
You defined the function as `loadScript(src, options)` but here you call it as if it were defined as `loadScript(src, callbackSuccess, callbackError)`.